docs/readme_template.md

```diff
@@ -0,0 +1,93 @@
+# Model Title
+
+### **Authors**
+*Anyone who should be attributed as part of the model. If multiple people or companies, use a comma seperated list*
+
+Example:
+
+Michael Zephyr, Dagaung Xu, Wenqi Li, NVIDIA
```

Comment:
There’s a typo in the names, perhaps let’s not put real names in this template file to avoid confusion.
```suggestion
Firstname1 LastName1, Firstname2 Lastname2, Affiliation1
```